### PR TITLE
Player: init player class

### PR DESCRIPTION
**Pull request by cchay**
Please review my code for the first draft of Player class.  I have a problem on line #94 where the code thinks I am passing in an argument, but it is not.  Not sure what's going on. 

Please help.

### CODE REVIEW

File: Player/Player.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+#		self.SetStrength( self.GenerateAttribute() )
+#		self.SetAgility( self.GenerateAttribute() )
+#		self.SetConstitution( self.GenerateAttribute() )
+		self.SetXP( 1 )
+		self.SetLevel( 1 )
+#		self.SetHP( self.GenerateHP() )
+		self.SetAC( 1 )
+		return self
+		
+	def GenerateAttribute( self ):
```
Comment (christopherchay) [inline Player/Player.py]: You were missing a 'self' argument. I showed you how to fix it earlier. That should do it.

Comment (cchay) [reply Player/Player.py]: OK.  I will try and resubmit.

**Comment by celestachay** [comment]
I do not know what to do!
:(

**Review by CalebChay — APPROVED** [review]
Well done!

**Review by christopherchay — APPROVED** [review]
Awesome! Looks good. :)

**Comment by EmmanuelChay** [comment]
o_0 wow

**Comment by CalebChay** [comment]
Hey! how'd it merge?

**Pull request merged**